SConstruct

```diff
@@ -451,6 +451,9 @@ if selected_platform in platform_list:
         CacheDir(scons_cache_path)
         print("Scons cache enabled... (path: '" + scons_cache_path + "')")
 
+    if (not env.msvc):
+        env.Append(CCFLAGS=['-Werror=return-type'])
```

Comment:
Could you move it further up where the warning levels are defined? https://github.com/bruvzg/godot/blob/be3230eb755f819f4902cd76163be1f3b2b706ed/SConstruct#L332-L355